Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM node:18 as builder
-
+FROM node:alpine
 WORKDIR /app
-
-# Update npm
-RUN npm install -g npm@latest
+COPY next.config.js .
+COPY package.json .
+COPY postcss.config.js .
+COPY tailwind.config.js .
+COPY tsconfig.json .
+ COPY yarn.lock .
 
 # Install app dependencies
-COPY ["package.json", "yarn.lock*", "./"]
+ #RUN npm install 
 RUN yarn install
```

Comment:
How would yarn be installed into the image?

Reply:
@Sanchitbajaj02 yarn can be installed like npm. Both are packages  . Yarn is faster that why I did yarn install